website/.eslintrc.json

```diff
@@ -5,11 +5,10 @@
     "es6": true,
     "node": true
   },
-  "plugins": ["prettier", "react", "import"],
+  "plugins": ["react", "import"],
```

Comment:
I'm open to a discussion about changing how we use prettier/eslint  -- but I think that should be it's own issue/PR, not rolled into this.

(FWIW, I think we're doing it in a non-conflicting way: https://github.com/prettier/eslint-plugin-prettier#recommended-configuration)